deployment.rb

```diff
@@ -89,7 +89,8 @@ def load_configuration()
     'pusher_application_key'      => 'fake_application_key',
     'pusher_application_secret'   => 'fake_application_secret',
     'videos_s3_bucket'            => 'videos.code.org',
-    'videos_url'                  => '//videos.code.org'
+    'videos_url'                  => '//videos.code.org',
+    'aws_region'                  => 'us-east-1'
```

Comment:
Currently all of our AWS infrastructure is provisioned in us-east-1, and (correct me if I'm mistaken here) I believe there are restrictions/extra costs associated with cross-region data transfer (e.g. reading an S3 bucket/DynamoDB table in one region from an EC2 instance in another) that we would like to avoid, so I think it's fine for us to keep this as an application-wide default for now (so things are provisioned to `us-east-1` unless otherwise specified) and either override this config and/or reconsider this default if/when we consider deploying to a mix of AWS regions in the future.